gen/generator.go

```diff
@@ -1358,6 +1358,9 @@ func (m *Model) SetClassProperties(classDetails interface{}) {
 				removedValidValuesList := GetValidValuesToRemove(m.PkgName, propertyName, m.Definitions)
 				for _, details := range propertyValue.(map[string]interface{})["validValues"].([]interface{}) {
 					validValue := details.(map[string]interface{})["localName"].(string)
+					if propertyName == "remoteCtxPcTag" || propertyName == "remotePcTag" {
+						validValue = "defaultValue"
+					}
```

Comment:
should we define a constant for this at the top for all the propertyName that have defaultValue as valid value, or should we expose this in the overwrite logic?